scripts/scil_compute_ihMT_maps.py

```diff
@@ -0,0 +1,358 @@
+# -*- coding: utf-8 -*-
+"""
+Computing ihMT and non-ihMT maps
+Make a more details description !
+"""
+
+import argparse
+import os
+
+from bids import BIDSLayout
+import nibabel as nib
+import numpy as np
+import scipy.ndimage
+
+from scilpy.io.utils import (add_overwrite_arg)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument("in_bids_folder",
+                   help='Path of the ihMT maps from bids folder')
+    p.add_argument('out_dir',
+                   help='Path to output folder')
+    p.add_argument('in_participant',
```

Comment:
This should be optional if you want to run it on a single subject or a couple of them using `nargs='+'`

Reply:
could be rename subject instead of participant.